be/src/olap/rowset/segment_reader.cpp

```diff
@@ -39,7 +39,6 @@ SegmentReader::SegmentReader(
         const std::vector<uint32_t>& used_columns,
         const std::set<uint32_t>& load_bf_columns,
         const Conditions* conditions,
-        const std::vector<ColumnPredicate*>* col_predicates,
```

Comment:
You should make you initialization order in accordance with variable declaration order in header file.

Reply:
yes, this modification is for this purpose to fix compile warning problems